modules/administration-guide/partials/con_backup-and-recovery.adoc

```diff
@@ -0,0 +1,14 @@
+[id="backup-and-recovery_{context}"]
+= Backup and recovery
+
+Backing up {prod-short} involves a combination of the following processes that back up respective data:
```

Comment:
```suggestion
Backing up {prod-short} involves a combination of the following processes that back up respective data:
```
It's not clear to me here what `respective` means in this context.

Reply:
Thanks. I meant `different`.